### ISSUE TITLE
Docker Option

### ISSUE THREAD

**Issue by dcplaya** [akx/ruuvidriver#8]
Since I do not use Hass.IO but I do run HA core in a docker, is it possible to dockerize RuuviDriver so I can continue to keep all my services in dockers? Keeps things isolated and easy to backup/migrate if something were to go wrong too.

**Comment by akx** [comment]
It should well be. I don't have a Dockerfile for it at hand, but I think you don't need much more than  `node:12-stretch`, maybe with some `bluez` dev tools. 

At runtime container will also need the `NET_CAP_RAW` capability to be able to talk dirty to Bluetooth.

If you give writing the Dockerfile a shot, PRs are welcome, of course!

**Comment by dcplaya** [comment]
Hmm.....I cant seem to get yarn to work. I cant even get it to run outside of Docker.

Cloned the git, installed `yarn` and then ran `yarn start` just like your docs say and I get a 
```
00h00m00s 0/0: : ERROR: [Errno 2] No such file or directory: 'start'
```

**Comment by akx** [comment]
I think you might have the wrong `yarn` – one is a JavaScript package manager, the other is... I think it has something to do with distributed computing?

**Comment by akx** [comment]
I added a basic dockerfile in db639b1 – you could give it a shot with something like
```
docker build -t ruuvidriver .
docker run -it ruuvidriver --cap-add NET_CAP_RAW -p 52020:52020
```

**Comment by dcplaya** [comment]
I gave it a try and when I run it I get 

```
➜  ruuvidriver git:(master) docker run -it ruuvidriver --cap-add NET_CAP_RAW -p 52020:52020
node: bad option: --cap-add
```

**Comment by akx** [comment]
Oop, my bad. I always forget the Docker run args must come before the image name.

```
docker run -it --cap-add NET_CAP_RAW -p 52020:52020 ruuvidriver 
```

**Comment by dcplaya** [comment]
Gave it another try, even tried running it priviledged

```
➜  ruuvidriver git:(master) docker run -it --privileged   -p 52020:52020 ruuvidriver
node: ../deps/uv/src/unix/poll.c:123: uv_poll_start: Assertion `!(((handle)->flags & (UV_HANDLE_CLOSING | UV_HANDLE_CLOSED)) != 0)' failed.
Aborted (core dumped)
➜  ruuvidriver git:(master) docker run -it --cap-add NET_CAP_RAW -p 52020:52020 ruuvidriver
docker: Error response from daemon: invalid CapAdd: unknown capability: "CAP_NET_CAP_RAW".
See 'docker run --help'.
```

**Comment by mixu-** [comment]
With some modifications (as below), I was able to get the docker container running on a Raspberry pi 4 running buster. However, ruuvidriver is not detecting any tags when run it like this.

I added libudev-dev and libusb-1.0-0-dev because I noticed those were needed in order to be able to run ruuvidriver on buster in the first place. Something not stated in the readme at the moment.

```
FROM node:12-buster

RUN apt-get update && \
    apt-get install -y libudev-dev libusb-1.0-0-dev && \
    rm -rf /var/lib/apt/lists/*

WORKDIR /app
ADD . /app

RUN yarn
ENV RUUVI_PORT 52020
EXPOSE ${RUUVI_PORT}
CMD node index.js
```

If I launch the container like so:
```docker run -it --rm --net=host ruuvidriver```

I get:
```
Listening on 52020
```
And just `{}` as the response from http://my_ip:52020/tags
